receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -150,6 +150,12 @@ metrics:
     unit: "1"
     gauge:
       value_type: int
+  container.cpu.online:
```

Comment:
What's the use case for this metric that can't be covered by counting MTS of `container.cpu.usage.percpu`?